src/commands.rs

```diff
@@ -752,6 +752,195 @@ implement_commands! {
+    ///     let x: RedisResult<Vec<String>> = con.geo_hash("my_gis", "Palermo");
+    ///     // x is vec!["sqc8b49rny0"]
+    ///
+    ///     let x: RedisResult<Vec<String>> = con.geo_hash("my_gis", &["Palermo", "Catania"]);
+    ///     // x is vec!["sqc8b49rny0", "sqdtr74hyu0"]
+    /// }
+    /// ```
+    fn geo_hash<K: ToRedisArgs, M: ToRedisArgs>(key: K, members: M) {
+        cmd("GEOHASH").arg(key).arg(members)
+    }
+
+    /// Return the positions of all the specified members of the geospatial
+    /// index represented by the sorted set at key.
+    ///
+    /// Every position is a pair of `(longitude, latitude)`. [`redis::geo::Coord`][1]
+    /// can be used to convert these value in a struct.
+    ///
+    /// [1]: ./geo/struct.Coord.html
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use redis::{Commands, RedisResult};
+    /// use redis::geo::Coord;
+    ///
+    /// fn get_position(con: &mut redis::Connection) {
+    ///     let x: RedisResult<Vec<Vec<f64>>> = con.geo_pos("my_gis", &["Palermo", "Catania"]);
+    ///     // x is [ [ 13.361389, 38.115556 ], [ 15.087269, 37.502669 ] ];
+    ///
+    ///     let x: Vec<Coord<f64>> = con.geo_pos("my_gis", "Palermo").unwrap();
+    ///     // x[0].longitude is 13.361389
+    ///     // x[0].latitude is 38.115556
+    /// }
+    /// ```
+    fn geo_pos<K: ToRedisArgs, M: ToRedisArgs>(key: K, members: M) {
+        cmd("GEOPOS").arg(key).arg(members)
+    }
+
+    /// Return the members of a sorted set populated with geospatial information
+    /// using [`geo_add`](#method.geo_add), which are within the borders of the area
+    /// specified with the center location and the maximum distance from the center
+    /// (the radius).
+    ///
+    /// Every item in the result can be read with [`redis::geo::RadiusSearchResult`][1],
+    /// which support the multiple formats returned by `GEORADIUS`.
+    ///
+    /// [1]: ./geo/struct.RadiusSearchResult.html
+    ///
+    /// ```rust,no_run
+    /// use redis::{Commands, RedisResult};
+    /// use redis::geo::{RadiusOptions, RadiusSearchResult, RadiusOrder, Unit};
+    ///
+    /// fn radius(con: &mut redis::Connection) -> Vec<RadiusSearchResult> {
+    ///     let opts = RadiusOptions::default().with_dist().order(RadiusOrder::Asc);
+    ///     con.geo_radius("my_gis", 15.90, 37.21, 51.39, Unit::Kilometers, opts).unwrap()
+    /// }
+    /// ```
+    fn geo_radius<K: ToRedisArgs>(
+        key: K,
+        longitude: f64,
+        latitude: f64,
+        radius: f64,
+        unit: geo::Unit,
+        options: geo::RadiusOptions
+    ) {
+        cmd("GEORADIUS")
+            .arg(key)
+            .arg(longitude)
+            .arg(latitude)
+            .arg(radius)
+            .arg(unit)
+            .arg(options)
+    }
+
+    /// This command is exactly like [`geo_radius`](#method.geo_radius) with the sole
+    /// difference that instead of taking, as the center of the area to query, a
```

Comment:
This whole "instead of" makes it very hard to understand IMO

Reply:
It is a copy of the [original description](https://redis.io/commands/georadiusbymember).

Do you prefer the [description in Lettuce](https://github.com/lettuce-io/lettuce-core/blob/c98108dd7fbe9bee8702e468efff4b085a976374/src/main/java/io/lettuce/core/api/reactive/RedisGeoReactiveCommands.java#L100-L101)?

>  Retrieve members selected by distance with the center of `member`. The member itself is always contained in the results.

Reply:
Sounds better, yes